### ISSUE TITLE
Readme is confusing

### ISSUE THREAD

**Issue by maxonfjvipon** [yegor256/together#13]
This is the code in readme:
```java
new Together(
  () -> {
    // do the job
    return true;
  }
)
```

It's not valid because `Together` accepts `Actions` which is generic function that returns `T` and accepts `int`.

So it should be:
```java
new Together(
  thread -> {
    // do the job
    return true;
  }
)
```

**Comment by yegor256** [comment]
@maxonfjvipon fixed in cde9dc6 better now?

**Comment by maxonfjvipon** [comment]
@yegor256 yes, thanks
